pylibs/plotting/HistogramPlotter.py

```diff
@@ -334,13 +383,16 @@ def drawHists2D(self):
             return
 
         for hist, sample in self.histosamples2D:
+            self.normalizer.normalize(hist, sample)
+
             title = hist.getName() + "_" + sample.name
             canvas = TCanvas(
-                title, title, self.config.canvas_size[0], self.config.canvas_size[1])
+                title, title, self.config.canvas_size[0], self.config.canvas_size[0])
```

Comment:
Is it correct that you're talking the width of the canvas twice here, instead of width and height?

Reply:
Yeah another quickfix from me because I wanted the canvas size to be squared as I have the same range on the x and y axes. I included another canvas size for 2D hists in the config now instead